### ISSUE TITLE
This project is supported? development will be? 

### ISSUE THREAD

**Issue by BergSoft** [frankban/django-endless-pagination#83]
A great app! But 2 years no change :( - I want that he did not die

**Comment by BertrandBordage** [comment]
Yeah, this project needs a new maintainer. @frankban took a lot of time a few years ago to make this project so powerful and easy to use. Unfortunately it seems he’s working for Canonical now and is probably not using Django anymore.

Someone therefore has to propose itself to become the new maintainer, then @frankban will add him/her as admin for the repository & the PyPI package.

**Comment by shtalinberg** [comment]
Do you have candidates to new maintainers ?

**Comment by BertrandBordage** [comment]
None, as far as I know :'(
@frankban, could you decide something please?

**Comment by ashwin31** [comment]
Hi @frankban we can manage this repository if you would like to give it to us. We are exclusive django development company from India.

**Comment by helenst** [comment]
@frankban Are you still looking for maintainers? It's a great app but is falling out of date with recent Django versions. I'd be happy to spend some time getting it up to date.

**Comment by spout** [comment]
Still no answer from @frankban, someone forked endless to work with Django 1.9 ?
Looked for other packages here: https://www.djangopackages.com/grids/g/pagination/
Also the most popular 'django-pagination' isn't recently updated :confused:

**Comment by shtalinberg** [comment]
Hi all. 
I hope that @frankban will not mind
I create new app https://github.com/shtalinberg/django-el-pagination 
django-el-pagination based on django-endless-pagination==2.0 
I will try to provide support of this application 
I am planning: 
- review the current pull requests from this repository 
- review the issues of this repository
- create start version 2.1 with fixed critical bugs for today (for example Django 1.9  )
- Add 2-5 contributors to help support the application
Pull requests are welcome.

**Comment by ashwin31** [comment]
@shtalinberg 
excellent move. Thanks for you support to the community. Please add one or two to manage the repository to keep it active.

**Comment by spout** [comment]
@shtalinberg nice ! Thank you !

**Comment by shtalinberg** [comment]
django-el-pagination==2.1.0 released
need 2-5 contributors to better maintenance django-el-pagination
